app/src/main/java/org/simple/clinic/patient/recent/RecentPatientEvents.kt

```diff
@@ -0,0 +1,8 @@
+package org.simple.clinic.patient.recent
+
+import org.simple.clinic.widgets.UiEvent
+import java.util.UUID
+
+data class RecentPatientItemClicked(val patientUuid: UUID) : UiEvent {
+  override val analyticsName = "Recent Patients: Item clicked"
```

Comment:
`Item clicked` -> `Patient clicked`? 

Item isn't natural to read when you're looking at analytics.

Reply:
`Recent Patients: Patient clicked`?

Reply:
yep?

Reply:
Reworded this commit.